### ISSUE TITLE
QA for https://github.com/microsoft/workshop-library/blob/main/full/regression-aml-designer/README.md

### ISSUE THREAD

**Issue by jlooper** [microsoft/workshop-library#81]
- [ ] All elements of the workshop are in place (README, workshop designer, solution if necessary, any images if necessary, slides, a call-to-action with a Learn module linked (usually in "Next Steps" area), a knowledge check (either in slides or directly linked to the workshop README) and an area to extend knowledge.
- [ ] The workshop can be done in its listed time frame
- [ ] Instructions are clear and well written
- [ ] The workshop is bug-free
- [ ] The workshop 'flows' well
- [ ] There are no stray auth keys or secrets embedded in the code

**Comment by CinnamonXI** [comment]
Doesn't have a video walkthrough yet

**Comment by jlooper** [comment]
@shwars will you be doing the TTT video?

**Comment by shwars** [comment]
@jlooper yes, do you think I should just record it, or we should schedule MLSA call? I think the former is better, because video quality will be better, and we won't have too many calls for MLSAs
